.github/workflows/build.yml

```diff
@@ -27,18 +27,24 @@ jobs:
           # Adding a filter? Check for the need to add to the outputs as well
           filters: |
             common:
-              - 'packages/**'
               - 'Docker/**'
               - '.github/**'
               - 'docker-compose.yaml'
             account:
-              - 'services/account/**'
+              - 'apps/account-api/**'
+              - 'apps/account-worker/**'
+              - 'libs/account-lib/**'
```

Comment:
My guess is that after refactoring we will consolidate most of these libs into a common one that all apps could use

Reply:
correct.